### DISCUSSION TITLE
Welcome to gpt-pilot Discussions!

### DISCUSSION THREAD

**Discussion by zvone187** [Pythagora-io/gpt-pilot#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by msocko420** [comment]
I am curious as to why when I look to run a separate code for translating it is looking for a yaml file but when the runshortgpt python prompt runs, the UI works no problem. Am I missing a step?

**Reply by zvone187** [reply]
Hey @msocko420, so sorry for the late reply - I missed your comment. I'm not sure what do you mean? What separate code for translation do you mean?

**Reply by lucas-0liveira** [reply]
It seems like you might be encountering a configuration issue. When running a separate code for translation, it's possible that the code is expecting a YAML file for some specific settings or configurations. However, when using the runshortgpt Python prompt, it might handle these configurations internally, which is why the UI works smoothly.

To address this, it's recommended to review the documentation or instructions provided with the code you're using for translation. Make sure you have all the necessary files and configurations in place, including any YAML files that may be required. If you're still facing issues, you may want to seek assistance from the developer or community forum associated with the code you're working with.

If you provide more details about the specific code or tool you're using, I might be able to offer more targeted advice.

**Reply by xacton** [reply]
👀 my eye open

**Reply by tristdrum** [reply]
@zvone187 please remove @travishall34's spam comments 🙏🏼

**Comment by nalbion** [comment]
The discord invitation in the README is no longer valid.  

https://discord.gg/FWnRZdCb

**Reply by d9vsky** [reply]
I confirm, the discord links in readme don't work. Yours as well unfortunately. 
@zvone187 could you please share a new invitation link?

**Reply by zvone187** [reply]
Hey guys, I updated it yesterday. Can you check now - it should work.

**Reply by d9vsky** [reply]
Worked. Thank you. 

https://discord.gg/HaqXugmxr9

**Comment by Ajayvirmoti** [comment]
It is quite amazing to use it, will follow-up for share new things.

**Reply by zvone187** [reply]
Awesome

**Comment by PatrickHolderness** [comment]
I receive this error when attempting to create a new project. any advice? i would love to try this out! using windows 10 with postgresql
```python
------------------ STARTING NEW PROJECT ----------------------←[0m
If you wish to continue with this project in future run:
←[1m←[32mpython main.py app_id=f91c1e2b-c0f2-4e8d-807e-67ee523e296b←[0m
←[1m←[32m--------------------------------------------------------------
←[0m
? What is the project name? spm


←[31m---------- GPT PILOT EXITING WITH ERROR ----------←[0m
Traceback (most recent call last):
  File "C:\Users\Patrick\documents\projects\gpt-pilot\pilot\main.py", line 87, in <module>
    project.start()
  File "C:\Users\Patrick\documents\projects\gpt-pilot\pilot\helpers\Project.py", line 83, in start
    self.project_manager.get_project_description()
  File "C:\Users\Patrick\documents\projects\gpt-pilot\pilot\helpers\agents\ProductOwner.py", line 39, in get_project_description
    self.project.args['name'] = clean_filename(ask_user(self.project, 'What is the project name?'))
  File "C:\Users\Patrick\documents\projects\gpt-pilot\pilot\prompts\prompts.py", line 58, in ask_user
    logger.info('Q: %s', question)
  File "C:\Python310\lib\logging\__init__.py", line 1477, in info
    self._log(INFO, msg, args, **kwargs)
  File "C:\Python310\lib\logging\__init__.py", line 1624, in _log
    self.handle(record)
  File "C:\Python310\lib\logging\__init__.py", line 1634, in handle
    self.callHandlers(record)
  File "C:\Python310\lib\logging\__init__.py", line 1696, in callHandlers
    hdlr.handle(record)
  File "C:\Python310\lib\logging\__init__.py", line 964, in handle
    rv = self.filter(record)
  File "C:\Python310\lib\logging\__init__.py", line 823, in filter
    result = f(record) # assume callable - will raise if not
  File "C:\Users\Patrick\documents\projects\gpt-pilot\pilot\logger\logger.py", line 35, in filter_sensitive_fields
    args = record.args.copy()
AttributeError: 'tuple' object has no attribute 'copy'
←[31m--------------------------------------------------←[0m
```

**Reply by nalbion** [reply]
It looks like this is fixed now

**Comment by Phea5antP1ucker** [comment]
I am a complete novice here, I'm in the early stages of learning very basic python. I'm using (Android Only) so please be gentle with me

**Reply by lucas-0liveira** [reply]
yes

**Reply by tharons** [reply]
how This app work

**Reply by tharons** [reply]
if someone knows than Also Guide me what I have to do

**Comment by lucas-0liveira** [comment]
What aspect of gpt-pilot do you find most promising for your applications or projects? Is there any specific functionality you're looking forward to exploring?

**Comment by Mouhamed03** [comment]
How to convert a python code on a Algorithme code

**Comment by Lunadog55** [comment]
👋

**Comment by Leopag07** [comment]
ogni volta che cerco di generare una nuova app mi da questo errore:
Traceback (most recent call last):

File "c:\Users\leopa\gpt pilot app\gpt-pilot\pilot\main.py", line 8, in

from dotenv import load_dotenv

ModuleNotFoundError: No module named 'dotenv'

come posso fare per risolverlo?

**Comment by wasimbijilihereIam** [comment]
Dei Just keep watching da, we will develop way faster, release and grow way faster than you! :) no hard feelings

**Comment by Dickhorseofbalgoria** [comment]
So, anyone here these days?

I was wondering if somebody could help me; I'm having a issue, in which when I start GptPilot to create anything, he stucks on the first developer task, always writing over and over again the same tasks, withouth finishing it and moving foward. I'm using OpenRouter API key with models like openchat/openchat-7b:free and mistral-7b-instruct:free. 

Anyone knows why is it stuck like that? That's related to the models I'm using? Is there any possible free alternative for it?

Anyways, thanks for the attention.

**Reply by Phea5antP1ucker** [reply]
Greetings! I noticed your post on GitHub regarding a snag with GptPilot – it's quite a pickle, isn't it? Here's a rundown of some avenues you might explore:

- **Code Examination:** Peer into the code that GptPilot is churning out. Is there a chance it's caught in a loop or repetitively doing the same thing without moving forward?
- **API Key Inspection:** Revisit your OpenRouter API key. Confirm it's equipped with the necessary permissions for GptPilot's requests. Also, ponder whether you've exceeded the free tier's usage cap.
- **Model Capabilities:** The complimentary models at your disposal might be lacking for intricate coding tasks. If feasible, test the waters with a model that's a notch more potent and see if it makes a difference.
- **Enhanced Context:** Furnishing GptPilot with ample context can do wonders! Decompose your tasks into smaller segments and elucidate your project's architecture.
- **Debugging Endeavors:** Integrate some print commands or logging into the code produced by GptPilot. This strategy can illuminate what's transpiring behind the scenes and pinpoint where the hang-up might be.

Regrettably, stumbling upon a fully free substitute that mirrors GptPilot's functionality is challenging. However, the realm of open-source language models or specialized code generation libraries might offer some solace, albeit with a steeper setup curve. If these suggestions steer you toward a solution, do tell.

Should you be inclined to divulge some code snippets or delve deeper into your project's details, I'm here to offer more targeted guidance!

**Comment by Orphaliman** [comment]
Is there possibly a way that Pythagora could have a sound that alerts you when it needs your input or approval?
